ansible/inventory/aws.aws_ec2.yml

```diff
@@ -0,0 +1,9 @@
+# Ansible AWS EC2 dynamic inventory plugin
+# https://docs.ansible.com/ansible/2.6/plugins/inventory/aws_ec2.html
```

Comment:
I see 2.5, that is version pinned in requirements, has this too - https://docs.ansible.com/ansible/2.5/plugins/inventory/aws_ec2.html - so ok.

Reply:
2.5 is minimum requirement, the actual installed is 2.6